Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -820,15 +822,6 @@ public async Task<SandboxedProcessPipExecutionResult> RunAsync(CancellationToken
             }
         }
 
-        private SharedOpaqueOutputLogger CreateSharedOpaqueOutputLoggerIfConfigured()

```

Comment:
this now uniformly happens in PipExecutor so that different ProcessPipExecutors don't have to do it separately themselves.